src/libs/actions/Policy.js

```diff
@@ -131,6 +131,41 @@ function createAndNavigate(name = '') {
     create(name).then(navigateToPolicy);
 }
 
+
+/**
+ * Delete the policy
+ *
+ * @param {String} [policyID]
+ * @param {Boolean} shouldGrowl
+ * @param {Boolean} shouldAutomaticallyReroute
+ * @returns {Promise}
+ */
+function deletePolicy(policyID = '', shouldGrowl = true, shouldAutomaticallyReroute = true) {
```

Comment:
Why is policyID optional? It shouldn't be since the backend needs a policyID to know what to delete.

Reply:
I don't see shouldGrowl and shouldAutomaticallyReroute passed when we call this method. If that's the case let's not preoptimize here and remove these two params from the method call and just do the growl and reroute in the method. In future if there is a scenario to not show growl or reroute then we'll update as needed.